config/processEimage.py

```diff
@@ -0,0 +1 @@
+config.calibrate.photoCal.doSelectUnresolved=False
```

Comment:
Using the conda installation of the Stack at NERSC, I get this when I run `processEimage.py` from this branch:
```
  processEimage.py image_repo/ --id visit=2226 --output output_repo --doraise --clobber-config
: Loading config overrride file '/global/u1/j/jchiang8/twinkles/obs_lsstSim/config/processEimage.py'
Traceback (most recent call last):
  File "/global/u1/j/jchiang8/twinkles/obs_lsstSim/bin/processEimage.py", line 25, in <module>
    ProcessEimageTask.parseAndRun()
  File "/project/projectdirs/lsst/lsstDM/Cori/conda-env/2016-04-15/LSST_STACK/opt/lsst/pipe_base/python/lsst/pipe/base/cmdLineTask.py", line 448, in parseAndRun
    parsedCmd = argumentParser.parse_args(config=config, args=args, log=log, override=cls.applyOverrides)
  File "/project/projectdirs/lsst/lsstDM/Cori/conda-env/2016-04-15/LSST_STACK/opt/lsst/pipe_base/python/lsst/pipe/base/argumentParser.py", line 436, in parse_args
    self._applyInitialOverrides(namespace)
  File "/project/projectdirs/lsst/lsstDM/Cori/conda-env/2016-04-15/LSST_STACK/opt/lsst/pipe_base/python/lsst/pipe/base/argumentParser.py", line 599, in _applyInit\
ialOverrides
    namespace.config.load(filePath)
  File "/project/projectdirs/lsst/lsstDM/Cori/conda-env/2016-04-15/LSST_STACK/opt/lsst/pex_config/python/lsst/pex/config/config.py", line 532, in load
    self.loadFromStream(stream=code, root=root)
  File "/project/projectdirs/lsst/lsstDM/Cori/conda-env/2016-04-15/LSST_STACK/opt/lsst/pex_config/python/lsst/pex/config/config.py", line 552, in loadFromStream
    exec stream in {}, local
  File "/global/u1/j/jchiang8/twinkles/obs_lsstSim/config/processEimage.py", line 1, in <module>
    config.calibrate.photoCal.doSelectUnresolved=False
  File "/project/projectdirs/lsst/lsstDM/Cori/conda-env/2016-04-15/LSST_STACK/opt/lsst/pex_config/python/lsst/pex/config/configurableField.py", line 120, in __set\
attr__
    self._value.__setattr__(name, value, at=at, label=label)
  File "/project/projectdirs/lsst/lsstDM/Cori/conda-env/2016-04-15/LSST_STACK/opt/lsst/pex_config/python/lsst/pex/config/config.py", line 692, in __setattr__
    raise AttributeError("%s has no attribute %s"%(_typeStr(self), attr))
AttributeError: lsst.pipe.tasks.photoCal.PhotoCalConfig has no attribute doSelectUnresolved
```